streamparse/ext/util.py

```diff
@@ -96,3 +96,9 @@ def get_nimbus_for_env_config(env_config):
         port = 6627
 
     return (host, port)
+
+
+def check_if_use_ssh_for_nimbus(env_config):
```

Comment:
Please rename `check_if_use_ssh_for_nimbus` to `is_ssh_for_nimbus` to stick to the convention of boolean functions starting with `is`.